parser.y

```diff
@@ -1382,7 +1383,7 @@ PlacementCountOpt:
 	}
 
 PlacementLabelOpt:
-	"LABEL" "=" stringLit
+	"CONSTRAINTS" "=" stringLit
```

Comment:
Question: what if we use `EqOpt` instead of `'='` here 🤔 

```sql
ALTER TABLE t ALTER PARTITION p ADD PLACEMENT POLICY CONSTRAINTS '+zone=zone1' ROLE LEADER REPLICAS 1
```

Reply:
It should be fine. As that `PlacementSpec` must start with one of unreserved keywords `CONSTRAINTS`, `ROLE` or `REPLICAS`... Empty should not have an effect. I just had a test with the above statement, and it is passed.
But I do not know if I should use `EqOpt` instead of `"="`.

Reply:
After some thoughts, I will stick to `"="`. `=` can be a separator, visually, which can help you read the SQL statement.